README.md

```diff
@@ -3,55 +3,74 @@
 ![Build](https://github.com/citizen428/methodfinder/workflows/Build/badge.svg)
 [![Gem Version](https://img.shields.io/gem/v/methodfinder.svg)](https://rubygems.org/gems/methodfinder)
 
-<!-- START doctoc generated TOC please keep comment here to allow auto update -->
-<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-
-- [Warning](#warning)
-- [Installation](#installation)
-- [Usage](#usage)
+<!-- toc -->
+
+- [NAME](#name)
+- [INSTALLATION](#installation)
+- [SYNOPSIS](#synopsis)
+- [DESCRIPTION](#description)
+  - [Warning](#warning)
+- [API](#api)
   - [MethodFinder.find](#methodfinderfind)
   - [Object#find_method](#objectfind_method)
     - [Blacklists](#blacklists)
   - [MethodFinder.find_classes_and_modules](#methodfinderfind_classes_and_modules)
   - [MethodFinder.find_in_class_or_module](#methodfinderfind_in_class_or_module)
-  - [Troubleshooting](#troubleshooting)
-- [Development](#development)
-- [Contributing](#contributing)
-- [License](#license)
+- [TROUBLESHOOTING](#troubleshooting)
+- [DEVELOPMENT](#development)
+- [CONTRIBUTING](#contributing)
+- [SEE ALSO](#see-also)
+  - [Gems](#gems)
+  - [Misc](#misc)
+- [VERSION](#version)
+- [AUTHOR](#author)
+- [LICENSE](#license)
 
-<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+<!-- tocstop -->
 
-This project was originally inspired by Smalltalk's Method Finder, but
-additional features were added over time.
+## NAME
 
-## Warning
+MethodFinder - a Smalltalk-like Method Finder for Ruby
 
-Common sense not included!
+## INSTALLATION
 
-While this gem should generally be safe to use, it's still better to be safe
-than sorry, so use this with caution and maybe not on production data.
+    $ gem install methodfinder
 
-This was initially written for the students of the core Ruby course on
-[RubyLearning](http://rubylearning.org), so it's generally not tested in a Rails
-console, just plain IRB/Pry.
+## SYNOPSIS
 
-## Installation
+```ruby
+Welcome to IRB. # or Pry
 
-Add this line to your application's Gemfile:
+>> "Hello, world!".find_method("hello-world")
+#=> ["String#parameterize"]
 
-```ruby
-gem 'methodfinder'
+>> %i[foo bar baz].find_method(:baz)
```

Comment:
Let's please not use "foo", "bar", "baz" examples.

Reply:
For context:

https://dev.to/kiarathedev/why-i-don-t-use-foo-bar-baz-1kf7
https://dev.to/warrend/no-more-foo-bar-and-baz-56j4
https://www.crondose.com/2018/04/foo-bar-definition-why-you-should-not-use-it/